app/lib/cms/s3.rb

```diff
@@ -35,7 +35,7 @@ def region
     end
 
     def credentials
-      config.slice(:access_key_id, :secret_access_key) if enabled?
+      config.slice(:access_key_id, :secret_access_key, :session_token) if enabled?
```

Comment:
I guess this `session_token` value will actually come from another place, instead of a config file, as the value should be dynamic...
Depending on how it goes, we might need a "dynamic_credentials" of sort of even perform a request to AWS ourselves to get the credentials.

Reply:
If there is a standard path for this token, then we can just detect it I guess.